src/main/java/com/onaio/steps/modelViewWrapper/HouseholdViewWrapper.java

```diff
@@ -41,7 +42,7 @@ public Household getHousehold(int nameViewId, int numberViewId, int commentsView
         TextView nameView = (TextView) activity.findViewById(nameViewId);
         TextView numberView = (TextView) activity.findViewById(numberViewId);
         String phoneNumber = numberView.getText().toString();
-        String currentDate = new SimpleDateFormat(Constants.DATE_FORMAT).format(new Date());
+        String currentDate = new SimpleDateFormat(Constants.DATE_FORMAT, Locale.ENGLISH).format(new Date());
```

Comment:
Could this have been configurable? let's say a different country wants the default to be in a different locale, the code will have to change.